portfolio-website/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
It looks like the portfolio website files are part of this git commit. I would delete the portfolio-website folder on this branch before merging this in.